src/commands/cmd_memory.c

```
@@ -11,6 +11,121 @@
+	// update number of nodes sampled
+	n_sampled_nodes += nodes_sample_size;
+
+	// sample each label
+	unsigned short n = GraphContext_SchemaCount(gc, SCHEMA_NODE);
+	for(LabelID l = 0; l < n; l++) {
+		Node node;
+		GrB_Index id;
+		Delta_MatrixTupleIter it;
+
+		// attach iterator to the current label matrix
+		Delta_Matrix L = Graph_GetLabelMatrix(g, l);
```

Cppcheck (2.10-2)

[error] Common realloc mistake

(memleakOnRealloc)